### PR TITLE
Fixed example in gallery to allow a two-dimensional mean

### PR DESCRIPTION
**Pull request by M87K452b**
## Motivation

Documentation fix
## Documentation Reference
 
https://docs.manim.community/en/stable/examples.html#threedsurfaceplot

## Testing Status

built html locally to verify changes are good and here are no errors.

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -579,9 +579,9 @@ Special Camera Settings
            def param_gauss(u, v):
                x = u
                y = v
-               d = np.sqrt(x * x + y * y)
-               sigma, mu = 0.4, 0.0
-               z = np.exp(-((d - mu) ** 2 / (2.0 * sigma ** 2)))
+               sigma, mu = 0.4, [0.0,0.0]
+               d = np.linalg.norm(np.array([x-mu[0], y-mu[1]]))
+               z = np.exp(-( d** 2 / (2.0 * sigma ** 2)))
```
Comment (Darylgolden) [inline docs/source/examples.rst:584]:
```suggestion
               z = np.exp(-(d ** 2 / (2.0 * sigma ** 2)))
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Hi, thank you for your contribution! I've left some suggestions to ensure the code matches our formatting style, please commit them.

File: docs/source/examples.rst [hunk 2]
```diff
@@ -579,9 +579,9 @@ Special Camera Settings
            def param_gauss(u, v):
                x = u
                y = v
-               d = np.sqrt(x * x + y * y)
-               sigma, mu = 0.4, 0.0
-               z = np.exp(-((d - mu) ** 2 / (2.0 * sigma ** 2)))
+               sigma, mu = 0.4, [0.0,0.0]
```
[the remark this answers is not in the document]
Comment (M87K452b) [reply docs/source/examples.rst]: The suggested changes have been made.

File: docs/source/examples.rst [hunk 3]
```diff
@@ -579,9 +579,9 @@ Special Camera Settings
...
+               d = np.linalg.norm(np.array([x-mu[0], y-mu[1]]))
```
[the remark this answers is not in the document]
Comment (M87K452b) [reply docs/source/examples.rst]: The suggested changes have been made.

File: docs/source/examples.rst (hunk 1 above)
Comment (M87K452b) [reply docs/source/examples.rst]: The suggested changes have been made.

**Review by Darylgolden — APPROVED** [review]
Thank you, this LGTM! In future you can just click the "add suggestion to batch" and then "commit suggestions" so that GitHub will do it for you.

File: docs/source/examples.rst [hunk 4]
```diff
@@ -579,9 +579,9 @@ Special Camera Settings
            def param_gauss(u, v):
                x = u
                y = v
-               d = np.sqrt(x * x + y * y)
-               sigma, mu = 0.4, 0.0
-               z = np.exp(-((d - mu) ** 2 / (2.0 * sigma ** 2)))
+               sigma, mu = 0.4, [0.0, 0.0]
+               d = np.linalg.norm(np.array([x - mu[0], y - mu[1]]))
```
Comment (kolibril13) [inline docs/source/examples.rst:583]: sounds good, but why not something like
sigma, mu_x ,mu_y = 0.4, 0.0, 0.0
d =np.sqrt( (x - mu_x)**2+ (y -mu_y)**2)

Comment (Darylgolden) [reply docs/source/examples.rst:583]: I think representing mu as a vector/coordinate makes more sense, because that's what it is.

Comment (kolibril13) [reply docs/source/examples.rst:583]: okay, then we can merge it like it is right now

**Review by kolibril13 — APPROVED** [review]

**Pull request merged**